Gists/ScoreAfterRetrieveAnomaly.cs

```diff
@@ -0,0 +1,36 @@
+﻿using BigML;
+using System;
+using System.Threading.Tasks;
+
+namespace Demo
+{
+    class ScoreAfterRetrieveAnomaly
+    {
+        static async void Main()
+        {
+            // New BigML client with username and API key
+            Console.Write("user: "); var User = Console.ReadLine();
+            Console.Write("key: "); var ApiKey = Console.ReadLine();
+            var client = new Client(User, ApiKey);
+
+            // retrieve a anomaly detector with a known ID
+            Anomaly anomaly;
+            string anomalyId = "anomaly/54daa82eaf447f5daa000XXY"; //Put your ID here
+            while ((anomaly = await client.Get<Anomaly>(anomalyId)).StatusMessage.StatusCode != Code.Finished) await Task.Delay(10);
```

Comment:
I know that the user has to check if the resource status is the right one (FINISHED) before trying to create a prediction, centroid or anomalyscore but make the code more complicate with the `while(....)` process is not worth.